build/charts/flow-aggregator/values.yaml

```diff
@@ -77,6 +78,27 @@ flowCollector:
   # default value will be used based on the protocol (tcp or udp) used to connect to the collector.
   # Min valid value is 512 and max valid value is 65535.
   maxIPFIXMsgSize: 0
+  # TLS / mTLS configuration when exporting to the flowCollector.
+  # This is only available when using tcp as the protocol.
+  tls:
+    # -- Enable TLS.
+    enable: false
+    # -- Name of the Secret containing the CA certificate used to authenticate the
+    # flowCollector. Default root CAs will be used if this field is empty. The certificate must
+    # contain the ca.crt key.
+    caSecretName: ""
```

Comment:
Better to keep the caCerts config and comment consistent ? I noticed the one used in clickhouse is:
```
    # -- Indicates whether to use custom CA certificate. Default root CAs will be used if this field is false.
    # If true, a Secret named "clickhouse-ca" must be provided with the following keys:
    # ca.crt: <CA certificate>
    caCert: false
```

Reply:
I spent a while thinking about this while working on the patch, and decided I preferred the ability to provide a Secret name in this case (rather than rely on a hardcoded name).